### PR TITLE
Add reference to Domain/SSL in "How To" README

### PR DESCRIPTION
**Pull request by cbortz**
## Changes

- Create "pointer" to external Domain/SSL doc
- Add reference to "pointer" Domain/SSL doc
- Convert format from reStructured Text to Markdown
- Reformat Development Workflow document
  + Uses clearer, simpler heading levels

### CODE REVIEW

**Comment by cbortz** [comment]
I think there may be some other documents that we want to create "pointers" for, and place behind our private repo.

File: how-to/domains-ssl-you.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# :rocket: Domains, SSL, & YOU!
+
+> This document is private and hosted in another repository. Please follow the link below to access it.
```
Comment (dleve123) [inline how-to/domains-ssl-you.md:3]: Why the blockquote? Doesn't seem necessary to me, but don't care too strongly

File: how-to/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## ... setup custom Domains w/ SSL?
+
+See: [":rocket: __Domains, SSL, & YOU!__"](domains-ssl-you.md)
+
+## ... start a new Rails app?
+
+Use [suspenders]:
+
+``` sourceCode
```
Comment (dleve123) [inline how-to/README.md]: I believe this is Bash, can we replace "``` sourceCode" with "```bash" – maybe I'm unfamiliar with this sourceCode thing though.

File: how-to/domains-ssl-you.md (hunk 1 above)
Comment (cbortz) [reply how-to/domains-ssl-you.md:3]: To me, it gives the feeling of something foreign or external. I don't have a strong opinion one way or the other.

File: how-to/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+$ gem install suspenders
+$ suspenders the-name-of-your-project-here
+$ cd the-name-of-your-project-here/
+$ bin/setup
+$ rake
+```
+
+## feature-test a Rails app’s Javascript?
```
Comment (dleve123) [inline how-to/README.md]: should we have some ...'s here

Comment (cbortz) [reply how-to/README.md]: ...probably... 👍

**Review by dleve123 — APPROVED** [review]
Left a couple of comments - I would consider those and potentially revise, but don't care too strongly about them. LGTM

File: how-to/README.md (hunk 2 above)
Comment (cbortz) [reply how-to/README.md]: I ran the file through a converter and completely missed that. Thanks!

**Pull request merged**